x-pack/plugins/observability/public/pages/slos/slos.test.tsx

```diff
@@ -283,7 +283,10 @@ describe('SLOs Page', () => {
 
         screen.getByTestId('confirmModalConfirmButton').click();
 
-        expect(mockDeleteSlo).toBeCalledWith({ id: sloList.results.at(0)?.id });
+        expect(mockDeleteSlo).toBeCalledWith({
+          id: sloList.results.at(0)?.id,
+          name: sloList.results.at(0)?.name,
```

Comment:
we now pass the `name` along to the hook, so it can display the name inside the Success / Error Toast message.